test/dataTypes.ts

```diff
@@ -0,0 +1,348 @@
+import * as assert from 'assert'
+import * as RLP from '../src'
+const BN = require('bn.js')
+
+export function runDataTypeTests() {
```

Comment:
why not put this function in separete file like `dataTypes.spec.ts`? The same could be done for rest of the files and then `test/index.ts` will not be needed. I think this would require small change to `package.json`:

```
    "unitTests": "mocha --reporter spec --require ts-node/register 'test/**.spec.ts'",

```

Reply:
Personally, I really like `.spec` / `.test` postfixes because you can quickly realize based on the filename what you're browsing. Also, it's crucial when you keep tests inside `src` dir.

That being said I know that my views can be very opinionated and I simply got used to doing thing this way, so I don't mind ;)

Reply:
Ok, especially the filename differentiation argument is really convincing, have updated this. Can imagine that this has other advantages on script runs e.g.